packages/desktop-client/src/components/sidebar.js

```diff
@@ -445,10 +445,7 @@ function Accounts({
 
 function ToggleButton({ style, isFloating, onFloat }) {
   return (
-    <View
-      className="float"
-      style={[style, { flexShrink: 0, color: colors.n5 }]}
-    >
+    <View className="float" style={[style, { flexShrink: 0 }]}>
       <Button type="bare" onClick={onFloat}>
```

Comment:
This should have the same effect:
```suggestion
    <View className="float" style={[style, { flexShrink: 0 }]}>
      <Button type="bare" onClick={onFloat} style={{ color: colors.n5 }}>
```

…without having to duplicate the color name

Reply:
Good tip! https://github.com/actualbudget/actual/pull/1421/commits/cd593440eeab2a40db03daa5754a4f92638382be